gwdetchar/omega/html.py

```diff
@@ -381,8 +303,7 @@ def close_page(page, target, about=None, date=None):
         `~datetime.datetime.now`
     """
     page.div.close()  # container
-    page.add(write_footer(about=about, date=date, class_=True,
-                          linkstyle='color:#eee;'))
+    page.add(io_html.write_footer(about=about, date=date))
```

Comment:
@duncanmmacleod, some of the unit tests are failing because this line needs to retain the `class_` and `linkstyle` kwargs:

```suggestion
    page.add(io_html.write_footer(about=about, date=date, class_=True,
                                  linkstyle='color:#eee;'))
```

`class_=True` will render the footer as a `footer` class, rather than a simple `div`, while the `linkstyle` is `'color:#000;'` by default (which is no good for omega scan footers).